### ISSUE TITLE
Cookiecutter failing if Windows Continuous Integration is declined

### ISSUE THREAD

**Issue by alexholehouse** [MolSSI/cookiecutter-cms#75]
Hi there,

Just in the last few weeks using cookiecutter with the MolSSI template fails for me when I decline Windows Continuous Integration (last step). Specific error is below;

```
Traceback (most recent call last):
  File "/var/folders/2n/xtzsyspd32v6vglg_pd5gmw80000gn/T/tmpvfncakrz.py", line 52, in <module>
    remove_windows_ci()
  File "/var/folders/2n/xtzsyspd32v6vglg_pd5gmw80000gn/T/tmpvfncakrz.py", line 49, in remove_windows_ci
    os.remove(os.path.join("devtools", "conda-recipe", "bld.bat"))
FileNotFoundError: [Errno 2] No such file or directory: 'devtools/conda-recipe/bld.bat'
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)
```

I can provide more info if needed, but I believe this is reproducible. If I accept Windows continuous integration everything works fine.

**Comment by dgasmith** [comment]
Ah yes. We removed that file, but didn't update the hook. Surprised our CI didn't catch this. Perhaps @janash could tackle this one?

**Comment by Lnaden** [comment]
I can fix it, it'll take a few minutes, I know where the issue is

**Comment by alexholehouse** [comment]
Thanks guys - hope you're well :-)

**Comment by dgasmith** [comment]
Should have been fixed in #76 on the day you raised this. Let me know if you still have an issue!
